From e6b2f220229815c0f179b38acefb8326248246ac Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 8 Dec 2021 14:54:27 +0100
Subject: [PATCH 0522/1695] plat-stm32mp1: update boot_api to include new magic
 numbers

Add the specific magic numbers to identify the low power
processing phases.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git be38a3cc9a24a22c611ffb396be0bc7602d313d6 plat-stm32mp1: update boot_api to include new magic numbers]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I1ef1c89046ab152c9a601bf9d341eea6c766ffcb
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/358365
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp1/boot_api.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/boot_api.h b/core/arch/arm/plat-stm32mp1/boot_api.h
index 62e38b588..a7daffd44 100644
--- a/core/arch/arm/plat-stm32mp1/boot_api.h
+++ b/core/arch/arm/plat-stm32mp1/boot_api.h
@@ -14,7 +14,9 @@
 #define BCKR_CORE1_MAGIC_NUMBER			4
 
 /* Value for BCKR_CORE1_MAGIC_NUMBER entry */
+#define BOOT_API_A7_CORE0_MAGIC_NUMBER		0xca7face0
 #define BOOT_API_A7_CORE1_MAGIC_NUMBER		0xca7face1
+#define BOOT_API_A7_RESET_MAGIC_NUMBER		0xca7dead0
 
 /* Backup register #5: physical address of core1 entry at boot up */
 #define BCKR_CORE1_BRANCH_ADDRESS		5
-- 
2.47.2

